src/script-loader.ts

```diff
@@ -27,6 +32,13 @@ export default class ScriptLoader {
             this._scripts[src] = new Promise((resolve, reject) => {
                 const script = document.createElement('script') as LegacyHTMLScriptElement;
                 const { async = false } = options || {};
+                const scriptAttributes = options && options.attributes;
+
+                for (const key in scriptAttributes) {
+                    if (scriptAttributes.hasOwnProperty(key)) {
```

Comment:
You can just use `Object.keys` so you don't need to do this check. i.e.:

```ts
Object.keys(attributes)
    .forEach(key => {
        script.setAttribute(key, attributes[key]);
    });
```

Reply:
Fixed